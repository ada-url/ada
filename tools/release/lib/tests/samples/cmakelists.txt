cmake_minimum_required(VERSION 3.16)

project(ada
  DESCRIPTION "Fast spec-compliant URL parser"
  LANGUAGES C CXX
  VERSION 1.0.0
)

set(ADA_LIB_VERSION "1.0.0" CACHE STRING "ada library version")
set(ADA_LIB_SOVERSION "1" CACHE STRING "ada library soversion")
