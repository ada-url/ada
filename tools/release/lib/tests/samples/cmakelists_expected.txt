cmake_minimum_required(VERSION 3.16)

project(ada
  DESCRIPTION "Fast spec-compliant URL parser"
  LANGUAGES C CXX
  VERSION 2.0.0
)

set(ADA_LIB_VERSION "2.0.0" CACHE STRING "ada library version")
set(ADA_LIB_SOVERSION "2" CACHE STRING "ada library soversion")
